docs/content.zh/docs/deployment/metric_reporters.md

```diff
@@ -24,31 +24,33 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Metric Reporters
+<a name="metric-reporters"></a>
 
-Flink allows reporting metrics to external systems.
-For more information about Flink's metric system go to the [metric system documentation]({{< ref "docs/ops/metrics" >}}).
+# 指标发送器
 
+Flink 支持用户将 Flink 的各项运行时指标发送给外部系统。
+了解更多指标方面信息可查看 [metric system documentation]({{< ref "zh/docs/ops/metrics" >}})。
```

Comment:
```suggestion
了解更多指标方面信息可查看 [metric system documentation]({{< ref "docs/ops/metrics" >}})。
```
```suggestion
了解更多指标方面信息可查看 [metric system documentation]({{< ref "zh/docs/ops/metrics" >}})。
```

Reply:
okay,I will fix that later.

Reply:
Hi @RocMarshal ,Is the reason using "docs/opt/metric" instead of "zh/docs/opt/metric" because of the [pr](https://github.com/apache/flink/pull/17902) is still not merged?

Reply:
You could run command `./build_docs.sh -p` in dir `flink/docs`, then observe the build log and click the links in the page to check the final link position.

It's necessary to explain that the prefix `zh/` is deprecated since the pages in the corresponding dir like `content` or `content.zh` with the language tag.

Reply:
Hi,@RocMarshal ,thanks for you update.
Do you mean  

> [DataStream API]({% link dev/datastream_api.md %}) 需要修改成中文链接
this part in the 
 [Flink+Translation+Specifications](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications) is a little bit out of date?

I add prefix `zh/` only because of this line.Oops.Maybe we should update the translation specifications later?

Reply:
Anyway,I did try runing ./build_docs.sh -p when I translated this pr,but the logs keep showing:

`(base) yangck@yangck-ThinkBook-14-G3-ACL:/data/flink_test/flink-orign/docs$ ./build_docs.sh -p
Building sites … ERROR 2022/03/23 23:25:57 [en] REF_NOT_FOUND: Ref "/docs/dev/datastream/execution_mode": "/data/flink_test/flink-orign/docs/content/docs/concepts/glossary.md:162:38": page not found
ERROR 2022/03/23 23:25:57 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/deployment/overview.md:65:34": page not found
ERROR 2022/03/23 23:25:57 [en] REF_NOT_FOUND: Ref "docs/deployment/resource-providers/standalone/": "/data/flink_test/flink-orign/docs/content/docs/deployment/overview.md:83:34": page not found
ERROR 2022/03/23 23:25:57 [en] REF_NOT_FOUND: Ref "docs/dev/datastream/execution_mode": "/data/flink_test/flink-orign/docs/content/docs/dev/dataset/overview.md:1:1": page not found
ERROR 2022/03/23 23:25:57 [en] REF_NOT_FOUND: Ref "docs/dev/python/datastream_tutorial": "/data/flink_test/flink-orign/docs/content/docs/dev/python/overview.md:1:1": page not found
ERROR 2022/03/23 23:25:57 [en] REF_NOT_FOUND: Ref "docs/dev/python/table_api_tutorial": "/data/flink_test/flink-orign/docs/content/docs/dev/python/overview.md:1:1": page not found
ERROR 2022/03/23 23:25:57 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/overview.md:61:16": page not found
WARN 2022/03/23 23:25:57 Expand shortcode is deprecated. Use 'details' instead.
ERROR 2022/03/23 23:25:57 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/gettingStarted.md:49:18": page not found
ERROR 2022/03/23 23:25:57 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/gettingStarted.md:160:16": page not found
ERROR 2022/03/23 23:25:57 [en] REF_NOT_FOUND: Ref "docs/dev/datastream/execution_mode": "/data/flink_test/flink-orign/docs/content/docs/libs/state_processor_api.md:1:1": page not found
ERROR 2022/03/23 23:25:58 [en] REF_NOT_FOUND: Ref "docs/dev/datastream/execution_mode": "/data/flink_test/flink-orign/docs/content/docs/dev/table/data_stream_api.md:658:108": page not found
ERROR 2022/03/23 23:25:58 [en] REF_NOT_FOUND: Ref "docs/dev/datastream/execution_mode": "/data/flink_test/flink-orign/docs/content/docs/ops/batch/blocking_shuffle.md:32:64": page not found
ERROR 2022/03/23 23:25:58 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/create.md:1:1": page not found
ERROR 2022/03/23 23:25:58 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/configuration/connector.md:1:1": page not found
ERROR 2022/03/23 23:25:58 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/drop.md:1:1": page not found
ERROR 2022/03/23 23:25:58 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/alter.md:1:1": page not found
ERROR 2022/03/23 23:25:58 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/insert.md:1:1": page not found
ERROR 2022/03/23 23:25:58 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/describe.md:1:1": page not found
ERROR 2022/03/23 23:25:58 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/explain.md:1:1": page not found
ERROR 2022/03/23 23:25:59 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/use.md:1:1": page not found
ERROR 2022/03/23 23:25:59 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/show.md:1:1": page not found
ERROR 2022/03/23 23:25:59 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/show.md:748:101": page not found
ERROR 2022/03/23 23:25:59 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/load.md:1:1": page not found
ERROR 2022/03/23 23:25:59 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/unload.md:1:1": page not found
ERROR 2022/03/23 23:25:59 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/set.md:1:1": page not found
ERROR 2022/03/23 23:25:59 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/reset.md:1:1": page not found
ERROR 2022/03/23 23:25:59 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/jar.md:37:93": page not found
ERROR 2022/03/23 23:25:59 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/jar.md:1:1": page not found
ERROR 2022/03/23 23:25:59 [en] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content/docs/dev/table/sql/queries/match_recognize.md:104:14": page not found
ERROR 2022/03/23 23:26:00 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/deployment/overview.md:65:34": page not found
ERROR 2022/03/23 23:26:00 [zh] REF_NOT_FOUND: Ref "docs/deployment/resource-providers/standalone/": "/data/flink_test/flink-orign/docs/content.zh/docs/deployment/overview.md:83:34": page not found
ERROR 2022/03/23 23:26:00 [zh] REF_NOT_FOUND: Ref "docs/dev/datastream/execution_mode": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/dataset/overview.md:1:1": page not found
ERROR 2022/03/23 23:26:00 [zh] REF_NOT_FOUND: Ref "docs/dev/python/datastream_tutorial": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/python/overview.md:1:1": page not found
ERROR 2022/03/23 23:26:00 [zh] REF_NOT_FOUND: Ref "docs/dev/python/table_api_tutorial": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/python/overview.md:1:1": page not found
ERROR 2022/03/23 23:26:00 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/overview.md:52:16": page not found
WARN 2022/03/23 23:26:00 Expand shortcode is deprecated. Use 'details' instead.
ERROR 2022/03/23 23:26:01 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/gettingStarted.md:49:17": page not found
ERROR 2022/03/23 23:26:01 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/gettingStarted.md:159:19": page not found
ERROR 2022/03/23 23:26:01 [zh] REF_NOT_FOUND: Ref "docs/dev/datastream/execution_mode": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/data_stream_api.md:658:108": page not found
ERROR 2022/03/23 23:26:01 [zh] REF_NOT_FOUND: Ref "docs/dev/datastream/execution_mode": "/data/flink_test/flink-orign/docs/content.zh/docs/libs/state_processor_api.md:1:1": page not found
ERROR 2022/03/23 23:26:01 [zh] REF_NOT_FOUND: Ref "docs/dev/datastream/execution_mode": "/data/flink_test/flink-orign/docs/content.zh/docs/ops/batch/blocking_shuffle.md:32:24": page not found
ERROR 2022/03/23 23:26:01 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/create.md:1:1": page not found
ERROR 2022/03/23 23:26:01 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/drop.md:1:1": page not found
ERROR 2022/03/23 23:26:01 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/configuration/connector.md:1:1": page not found
ERROR 2022/03/23 23:26:01 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/alter.md:1:1": page not found
ERROR 2022/03/23 23:26:01 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/insert.md:1:1": page not found
ERROR 2022/03/23 23:26:02 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/describe.md:1:1": page not found
ERROR 2022/03/23 23:26:02 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/explain.md:1:1": page not found
ERROR 2022/03/23 23:26:02 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/show.md:1:1": page not found
ERROR 2022/03/23 23:26:02 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/show.md:749:94": page not found
ERROR 2022/03/23 23:26:02 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/load.md:1:1": page not found
ERROR 2022/03/23 23:26:02 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/unload.md:1:1": page not found
ERROR 2022/03/23 23:26:02 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/set.md:1:1": page not found
ERROR 2022/03/23 23:26:02 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/reset.md:1:1": page not found
ERROR 2022/03/23 23:26:02 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/jar.md:38:81": page not found
ERROR 2022/03/23 23:26:02 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/jar.md:1:1": page not found
ERROR 2022/03/23 23:26:02 [zh] REF_NOT_FOUND: Ref "docs/dev/table/sqlClient": "/data/flink_test/flink-orign/docs/content.zh/docs/dev/table/sql/queries/match_recognize.md:89:30": page not found
Built in 6554 ms
Error: Error building site: logged 56 error(s)
`

And it just cannot run and nothing shows on http://localhost:1313/ 

Anyway, I'm using Linux Mint 20 Do you get the same error? or should I create a jira to report this?
@RocMarshal

Reply:
> the prefix `zh/` is deprecated

The prefix `zh/` is deprecated , so there is no need to update the translation specifications.

Reply:
Okay,thanks @RocMarshal ,I will remove prefix and figure out the hugo things first.